bolt/lib/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -1321,17 +1321,49 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
 
   int getUncondBranchEncodingSize() const override { return 28; }
 
+  // This helper function creates the snippet of code
+  // that compares a register RegNo with an immedaite Imm,
+  // and jumps to Target if they are equal.
```

Comment:
nit: please format using 80-column width. Same for the other comment.

Reply:
I will fix the comments.